src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -93,6 +104,50 @@ export default {
         }
     },
     methods: {
+        setMoveIndex(index) {
```

Comment:
You don't need this method. See comment above about not using the `set-move-index` event.

Reply:
N/A - leave as is for now.